ubuntu24.04/nvidia-driver

```diff
@@ -420,18 +371,29 @@ _unload_driver() {
 _install_driver() {
     local install_args=()
 
-    echo "Installing NVIDIA driver kernel modules..."
-    cd /usr/src/nvidia-${DRIVER_VERSION}
-    if [ -d /lib/modules/${KERNEL_VERSION}/kernel/drivers/video ]; then
-        rm -rf /lib/modules/${KERNEL_VERSION}/kernel/drivers/video
-    else
-        rm -rf /lib/modules/${KERNEL_VERSION}/video
-    fi
```

Comment:
Question -- is it safe to remove this? I am unfamiliar with why this logic was required.

Reply:
I had to remove this. It wouldn't work otherwise